show/fgnhg.py

```diff
@@ -41,7 +41,7 @@ def active_hops(nhg):
     if nhg is None:
         for nhg_prefix in table_keys:
             t_dict = state_db.get_all(state_db.STATE_DB, nhg_prefix)
-            vals = sorted(set([val for val in list(t_dict.values())]))
+            vals = sorted(set([val for val in t_dict.values()]))
```

Comment:
```suggestion
            vals = sorted(t_dict.values())
```

Reply:
Addressed in latest commit.

Reply:
FYI, I needed to change this back, because the conversion to a set eliminates duplicate values. Unit tests were failing.